tdrs-backend/tdpservice/users/authentication.py

```diff
@@ -12,12 +13,22 @@ class CustomAuthentication(BaseAuthentication):
     def authenticate(username=None, login_gov_uuid=None, hhs_id=None):
         """Authenticate user with the request and username."""
         User = get_user_model()
-
+        logging.debug("CustomAuthentication::authenticate:hhs_id {}".format(hhs_id))
+        logging.debug("CustomAuthentication::authenticate:login_gov_uuid {}".format(login_gov_uuid))
+        logging.debug("CustomAuthentication::authenticate:username {}".format(username))
         try:
-            if login_gov_uuid:
-                return User.objects.get(login_gov_uuid=login_gov_uuid)
-            elif hhs_id:
+            if hhs_id:
+                try:
+                    return User.objects.get(hhs_id=hhs_id)
+                except User.DoesNotExist:
```

Comment:
This will be the common use-case before we have a significant ACF user base as it will update previously-existing users who used our old login.gov solution with the new `hhs_id` value.